R/redcap_dates.R

```diff
@@ -0,0 +1,80 @@
+#' REDCap Date Conversion
+#'
+#' This function is trying to take all (most)
+#' possibilities to enter dates into account and converts them to as.Date
+#' format.
+#'
+#' @param var variable to convert
+#'
+#' @return converted variable
+#' @export
+#' @importFrom crayon bold underline red
+
+#'
+#' @examples
+#' var <- "dataframe$var"
```

Comment:
is this valid syntax? It'll error (or produce just NA), wont it? why not pass in a vector of potential dates `var <-c("01.12.2022", "12.2022", "2022", "01/12/2022")` then the user can see what the function actually does...

Reply:
done